doc/user/content/sql/alter-set-in-cluster.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "ALTER SET IN CLUSTER"
+description: "`ALTER SET IN CLUSTER` changes the association of an object to a cluster"
+menu:
+  main:
+    parent: 'commands'
+---
+
+`ALTER SET IN CLUSTER` changes the association of an object to a cluster.
+
+{{< private-preview />}}
+
+## Syntax
+
+{{< diagram "alter-set-in-cluster.svg" >}}
+
+## Examples
+
+### Materialized view
+
+Move a materialized view to a specific cluster:
+
+```sql
+ALTER MATERIALIZED VIEW mv1 SET IN CLUSTER c1;
+```
+
+## Privileges
+
+The privileges required to execute this statement are:
+
+- Ownership of the object.
+- Ownership of the cluster.
```

Comment:
The code requires `CREATE` on the new cluster, not ownership. We should also require `DELETE` from the old cluster.

Reply:
There's no explicit `DELETE` privilege. To drop an object, you must own that object, so that's captured in `Ownership of the object`.

I agree on the `CREATE` privileges.